src/modules/user/schema/user.schema.ts

```diff
@@ -6,28 +6,28 @@ export type UserDocument = User & Document;
 
 @Schema({ timestamps: true })
 export class User {
-  @Prop()
+  @Prop({ default: '' })
```

Comment:
Why these changes? The create user dto is already imposing the the requirements of filling those fields.

Reply:
And create dto is not needed but update dto is needed